wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -1206,6 +1206,12 @@ class SwiftGenerator private constructor(
       if (typeName is DeclaredTypeName) {
         imports.add(typeName.moduleName)
       }
+      if (field.isRepeated && typeName is ParameterizedTypeName) {
+        val declaredTypeName = typeName.typeArguments[0]
```

Comment:
Maybe do a for loop on typeArguments instead of just [0] ?

Reply:
I see this pattern been used everywhere else regarding `ParametrizedTypeName`

Reply:
OK yes this works for maps too so updated to loop through. Test case was added and verified it was broken and fixed with my change.